HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # Version History
 
+## 1.0.18 / 2021-09-29
+
+- Update pipelines to use Azure Key Vault for secrets
+
```

Comment:
Same comment here about noting the breaking change of client key to client secret in the config

Reply:
Updated history